### DISCUSSION TITLE
Profiler?

### DISCUSSION THREAD

**Discussion by hrose26** [AprilSylph/XKit-Rewritten#1120] [Suggestion Box]
I really miss the Profiler feature from New Xkit, it was incredibly helpful keeping track of who I was following. Such a tool would be really useful now since I mass followed a lot of people/artists after YET ANOTHER twitter scare. Is there any chance of it returning?

**Comment by AprilSylph** [comment]
Probably not, sorry.

**Comment by marcustyphoon** [comment]
Hmm. I guess in theory I could unbreak the one in the old version of xkit, but we don't have anyone to review it, really... and no, actually, that's a bad idea in case chrome removes support for the old version entirely; never mind.

**Comment by nightpool** [comment]
What specific parts of the Profiler feature were helpful for you? I only
have a vague memory of what the old one did and it would be helpful to hear
it in your own words

**Reply by hrose26** [reply]
I would give my mutuals nicknames next to their profile names on the dashboard. This could include the name I call them by, or even their pronouns. It's especially helpful in the roleplay community, when I was a part of it at least.
